app/presenters/school_presenter.rb

```diff
@@ -3,6 +3,20 @@ def location
     [model.name, model.town, model.county].reject(&:blank?).join(', ')
   end
 
+  def school_size
+    if model.gias_data.present?
+      return model.gias_data['NumberOfPupils'] + ' pupils enrolled' if model.gias_data['NumberOfPupils'].present?
+      return 'Up to ' + model.gias_data['SchoolCapacity'] + 'pupils' if model.gias_data['SchoolCapacity'].present?
+    end
+    'No data available'
```

Comment:
Should this 'No data available' come from en.yml?

Reply:
Yep! Good catch 😄

Reply:
Can we use string interpolation here?